test/test_scanner.py

```diff
@@ -480,6 +480,20 @@ def _file_test(self, url, filename, package, version):
                     "CVE-2000-0217",
                 ],
             ),
+            (
+                "test-wireshark-1.10.12.out",
```

Comment:
Alphabetize (as best you can, this section's a little messed up right now, but wireshark definitely shouldnt' go before png)